is_core/utils/field_api.py

```diff
@@ -85,8 +85,10 @@ class ModelFieldDescriptor(FieldDescriptor):
     def get_label(self):
         if self.model_field_or_method.auto_created and self.model_field_or_method.is_relation:
             return self.model_field_or_method.field.reverse_verbose_name
-        else:
+        elif hasattr(self.model_field_or_method, 'verbose_name'):
```

Comment:
generic FK field has no verbose_name but we have custom generic field in core which adds it